scripts/pre_push_hook.py

```diff
@@ -328,36 +329,52 @@ def install_hook():
         raise ValueError(err_chmod_cmd)
 
 
-def does_diff_include_js_or_ts_files(files_to_lint):
+def does_diff_include_js_or_ts_files(diff_files):
     """Returns true if diff includes JavaScript or TypeScript files.
 
     Args:
-        files_to_lint: list(str). List of files to be linted.
+        diff_files: list(str). List of files to be linted.
```

Comment:
Should we update the arg description accordingly? (here and elsewhere)

Reply:
done